tests/test_utils.py

```diff
@@ -33,14 +33,21 @@ def test_pyproject_toml_matches_version():
     "test_input, expected",
     [
         ("abcde:12345", "12345"),
-        ("abcde:my-name", "my_name"),
+        ("abcde:my-name", None),
```

Comment:
I think I misunderstood this when I first looked at it. since `my-name` is neither `alpha_user_my-name`, `airflow_blah`, `my-name@digital` or digits like the restricted admin role, it's considered invalid

Reply:
Yes, that's the idea - do you think it needs to allow for this case?